src/components/navigation-header/components/product-page-content/utils/get-nav-items.ts

```diff
@@ -33,11 +33,12 @@ export function getNavItems(currentProduct: ProductData): NavItem[] {
 			label: 'Tutorials',
 			url: `/${currentProduct.slug}/tutorials`,
 		},
-		{
+	]
+	currentProduct.slug !== 'hcp' &&
+		items.push({
 			label: 'Install',
 			url: `/${currentProduct.slug}/downloads`,
-		},
-	]
+		})
```

Comment:
nit: let's make this an `if` block instead of an expression:
```ts
if (currentProduct.slug !== 'hcp') {
  items.push({
    label: 'Install',
    url: `/${currentProduct.slug}/downloads`,
  })
}
```